### PR TITLE
[EraVM] Remove setRequiresStructuredCFG

### PR DESCRIPTION
**Pull request by vladimirradosavljevic**

### CODE REVIEW

File: llvm/lib/Target/EraVM/EraVMInstrInfo.cpp [hunk 1]
```diff
@@ -249,6 +249,11 @@ void EraVMInstrInfo::anchor() {}
 EraVMInstrInfo::EraVMInstrInfo()
     : EraVMGenInstrInfo(EraVM::ADJCALLSTACKDOWN, EraVM::ADJCALLSTACKUP), RI() {}
 
+unsigned int
+EraVMInstrInfo::getTailDuplicateSize(CodeGenOpt::Level OptLevel) const {
+  return 2;
```
Comment (akiramenai) [inline llvm/lib/Target/EraVM/EraVMInstrInfo.cpp]: Could we clarify?

**Review by akiramenai — APPROVED** [review]

File: llvm/lib/Target/EraVM/EraVMInstrInfo.cpp (hunk 1 above)
Comment (vladimirradosavljevic) [reply llvm/lib/Target/EraVM/EraVMInstrInfo.cpp]: Yes, I will add comment. I tried with 2, 4, 6, 10, let me try with 3 to check the numbers.

Comment (vladimirradosavljevic) [reply llvm/lib/Target/EraVM/EraVMInstrInfo.cpp]: Tail duplication size 3 for -O3 reduced worst real time cycles from -1.316% to -0.375%, so I think we should keep it. Wdyt?

Comment (akiramenai) [reply llvm/lib/Target/EraVM/EraVMInstrInfo.cpp]: Ok, let's keep it. Please document size values we experimented with.

**Pull request merged**